exporters/otlp/src/recordable.cc

```diff
@@ -159,6 +159,39 @@ void Recordable::SetName(nostd::string_view name) noexcept
   span_.set_name(name.data(), name.size());
 }
 
+void Recordable::SetSpanKind(opentelemetry::trace::SpanKind span_kind) noexcept
+{
+  opentelemetry::proto::trace::v1::Span_SpanKind proto_span_kind =
+      opentelemetry::proto::trace::v1::Span_SpanKind::Span_SpanKind_SPAN_KIND_UNSPECIFIED;
+
+  if (span_kind == opentelemetry::trace::SpanKind::kInternal)
```

Comment:
Nit: Is there a reason to use if/else over switch here?

Reply:
No particular reason. Have switched to `switch` now.